engine/flink/process/src/main/scala/pl/touk/nussknacker/engine/process/util/StateConfiguration.scala

```diff
@@ -15,7 +14,9 @@ object StateConfiguration {
 
   }
 
-  case class RocksDBStateBackendConfig(checkpointDataUri: String,
+  case class RocksDBStateBackendConfig(enable: Boolean,
+                                       // deprecated - Not needed after Flink 1.13
```

Comment:
why don't we just remove it?

Reply:
added more descriptive comment (I hope so)